tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -118,15 +118,10 @@ def run_pipeline_test(self, speech_recognizer, examples):
                 },
             )
         else:
-            # Non CTC models cannot use chunk_length
-            with self.assertRaises(ValueError) as v:
-                outputs = speech_recognizer(audio, chunk_length_s=10)
-            self.assertEqual(v.exception, "")
-
             # Non CTC models cannot use return_timestamps
             with self.assertRaises(ValueError) as v:
                 outputs = speech_recognizer(audio, return_timestamps="char")
-            self.assertEqual(v.exception, "")
+            self.assertEqual(str(v.exception), "We cannot return_timestamps yet on non-ctc models !")
```

Comment:
I don't understand why we expected empty string here, while we have

https://github.com/huggingface/transformers/blob/67d32f4649715db5cbac66deb759f9969dc1ff70/src/transformers/pipelines/automatic_speech_recognition.py#L405

@Narsil

Reply:
Nice catch. How was this not caught earlier ?

And for the empty string, it seems likely I forgot to update it.
Usually in my workflow, I create the failing test, do the scaffolding on how the test should look like, and updte the values when the test fails.

It seems I forgot to run/update this one.

Reply:
> How was this not caught earlier ?

I only found this after trying to use new tiny models. Previously, `Speech2TextConfig` would not generate a pipeline test (yeah, you know, there were a lot of reasons where a tiny model wouldn't be created). In the new tiny model creation script, I made more efforts to create those tiny models as many as possible.

> And for the empty string, it seems likely I forgot to update it.
No problem :-) I also guess so